challenge-06/challenge-06.js

```diff
@@ -32,19 +33,28 @@ Crie uma função chamada `showTeamPosition` com as seguintes características:
     - Se não houver time para a posição passada, deve retornar a mensagem:
     "Não temos a informação do time que está nessa posição."
 */
-// ?
+function showTeamPosition(x) {
+	return x <= 5 && x > 0 ? 'O time que está em ' + x + 'º lugar é o ' + teams[x - 1] + '.' : 'Não temos a informação do time que está nessa posição.';
```

Comment:
-1
Procure usar ternário somente quando a validação for pequena. Para frases maiores, que ficam muito "horizontais", pode usar um `if` mesmo, ou tentar quebrar o ternário nas linhas de baixo :)